### PR TITLE
zh-translation: /news/releases/1.0.x/announcing-1.0.3/index.md #1530

### PR DESCRIPTION
**Pull request by gorda**
Ref: [ServiceMesher#1530](https://github.com/servicemesher/istio-official-translation/issues/1530)

### CODE REVIEW

File: content/zh/news/releases/1.0.x/announcing-1.0.3/index.md [hunk 1]
```diff
@@ -12,34 +12,34 @@ aliases:
...
+- 现在，在配置 DNS 解析时，[Service entry](/zh/docs/reference/config/networking/service-entry/) 验证会拒绝通配主机名（`*`）。API 从未允许这样做，只是在以前的版本中，`ServiceEntry` 被错误地排除在验证之外。请将通配符作为主机名的一部分，例如 `*.bar.com`，可以保持不变。
 
-- The core dump path for `istio-proxy` has changed to `/var/lib/istio`.
+- `istio-proxy` 的核心转储路径已更改为 `/var/lib/istio`。
 
-## Networking
+## 网络{#networking}
 
-- [Mutual TLS](/zh/docs/tasks/security/authentication/mutual-tls) Permissive mode is enabled by default.
+- [双向 TLS](/zh/docs/tasks/security/authentication/mutual-tls) 许可模式现在是默认启用的。
```
Comment (ilylia) [inline content/zh/news/releases/1.0.x/announcing-1.0.3/index.md]:
```suggestion
- [双向 TLS](/zh/docs/tasks/security/authentication/mutual-tls) 宽容模式现在是默认启用的。
```

File: content/zh/news/releases/1.0.x/announcing-1.0.3/index.md [hunk 2]
```diff
@@ -12,34 +12,34 @@ aliases:
...
+- Mixer 客户端（`istio-policy`）现在支持 `FAIL_OPEN` 设置。
 
-- Istio Performance dashboard added to Grafana.
+- Istio 性能仪表盘已添加至 Grafana。
 
-- Reduced `istio-telemetry` CPU usage by 10%.
+- `istio-telemetry` CPU 使用率降低 10%。
 
-- Eliminated `statsd-to-prometheus` deployment. Prometheus now directly scrapes from `istio-proxy`.
+- 淘汰 `statsd-to-prometheus` deployment。现在可以直接从 `istio-proxy` 中抓取 Prometheus。
```
Comment (ilylia) [inline content/zh/news/releases/1.0.x/announcing-1.0.3/index.md]:
```suggestion
- 淘汰 `statsd-to-prometheus` deployment。Prometheus 现在可以直接从 `istio-proxy` 中抓取指标。
```

File: content/zh/news/releases/1.0.x/announcing-1.0.3/index.md [hunk 3]
```diff
@@ -12,34 +12,34 @@ aliases:
...
-We're pleased to announce the availability of Istio 1.0.3. Please see below for what's changed.
+我们很高兴的宣布 Istio 1.0.3 现已正式发布。下面是更新详情。
 
 {{< relnote >}}
 
-## Behavior changes
+## 行为变更{#behavior-changes}
 
-- [Validating webhook](/zh/docs/ops/common-problems/validation) is now mandatory. Disabling it may result in Pilot crashes.
+- 现在必须 [验证 webhook](/zh/docs/ops/common-problems/validation)。禁用它可能导致 Pilot 崩溃。
```
[the remark this answers is not in the document]
Comment (gorda) [reply content/zh/news/releases/1.0.x/announcing-1.0.3/index.md]: 已修改

File: content/zh/news/releases/1.0.x/announcing-1.0.3/index.md [hunk 4]
```diff
@@ -12,34 +12,34 @@ aliases:
...
 
-- [Service entry](/zh/docs/reference/config/networking/service-entry/) validation now rejects the wildcard hostname (`*`) when configuring DNS resolution. The API has never allowed this, however `ServiceEntry` was erroneously excluded from validation in the previous release. Use of wildcards as part of a hostname, e.g. `*.bar.com`, remains unchanged.
+- 现在，在配置 DNS 解析时，[Service entry](/zh/docs/reference/config/networking/service-entry/) 验证会拒绝通配主机名（`*`）。API 从未允许这样做，只是在以前的版本中，`ServiceEntry` 被错误地排除在验证之外。请将通配符作为主机名的一部分，例如 `*.bar.com`，可以保持不变。
```
[the remark this answers is not in the document]
Comment (gorda) [reply content/zh/news/releases/1.0.x/announcing-1.0.3/index.md]: 已修改

File: content/zh/news/releases/1.0.x/announcing-1.0.3/index.md (hunk 1 above)
Comment (gorda) [reply content/zh/news/releases/1.0.x/announcing-1.0.3/index.md]: 已修改

File: content/zh/news/releases/1.0.x/announcing-1.0.3/index.md (hunk 2 above)
Comment (gorda) [reply content/zh/news/releases/1.0.x/announcing-1.0.3/index.md]: 已修改，感谢您耐心的 review

image

**Pull request merged**